probe-rs/targets/CC13XX_CC26XX_Series.yaml

```diff
@@ -0,0 +1,284 @@
+  - !Nvm
+      name: ~
+      range:
+        start: 0
+        end: 0x58000
+      is_boot_memory: true
+      cores:
+      - main
+  flash_algorithms:
+  - cc13x2-cc26x2-flash
+- name: CC2652P
+  part: ~
+  cores:
+  - name: main
+    type: armv7em
+    core_access_options:
+      !Arm
+        ap: 0
+        psel: 0
+        debug_base: ~
+        cti_base: ~
+  jtag:
+    scan_chain:
+      - name: core0
+        ir_len: 4
+      - name: ICEPICK
+        ir_len: 6
+  memory_map:
+  - !Ram
+      name: ~
+      range:
+        start: 0x20000000
+        end: 0x20014000
+      is_boot_memory: false
+      cores:
+      - main
+  - !Nvm
+      name: ~
+      range:
+        start: 0
+        end: 0x58000
+      is_boot_memory: true
+      cores:
+      - main
+  flash_algorithms:
+  - cc13x2-cc26x2-flash
+- name: CC1312R
+  part: ~
+  cores:
+  - name: main
+    type: armv7em
+    core_access_options:
+      !Arm
+        ap: 0
+        psel: 0
+        debug_base: ~
+        cti_base: ~
+  jtag:
+    scan_chain:
+      - name: core0
+        ir_len: 4
+      - name: ICEPICK
+        ir_len: 6
+  memory_map:
+  - !Ram
+      name: ~
+      range:
+        start: 0x20000000
+        end: 0x20014000
+      is_boot_memory: false
+      cores:
+      - main
+  - !Nvm
+      name: ~
+      range:
+        start: 0
+        end: 0x58000
+      is_boot_memory: true
+      cores:
+      - main
+  flash_algorithms:
+  - cc13x2-cc26x2-flash
+- name: CC1352R
+  part: ~
+  cores:
+  - name: main
+    type: armv7em
+    core_access_options:
+      !Arm
+        ap: 0
+        psel: 0
+        debug_base: ~
+        cti_base: ~
+  jtag:
+    scan_chain:
+      - name: core0
+        ir_len: 4
+      - name: ICEPICK
+        ir_len: 6
+  memory_map:
+  - !Ram
+      name: ~
+      range:
+        start: 0x20000000
+        end: 0x20014000
+      is_boot_memory: false
+      cores:
+      - main
+  - !Nvm
+      name: ~
+      range:
+        start: 0
+        end: 0x58000
+      is_boot_memory: true
+      cores:
+      - main
+  flash_algorithms:
+  - cc13x2-cc26x2-flash
+- name: CC1352P
+  part: ~
+  cores:
+  - name: main
+    type: armv7em
+    core_access_options:
+      !Arm
+        ap: 0
+        psel: 0
+        debug_base: ~
+        cti_base: ~
+  jtag:
+    scan_chain:
+      - name: core0
+        ir_len: 4
+      - name: ICEPICK
+        ir_len: 6
+  memory_map:
+  - !Ram
+      name: ~
+      range:
+        start: 0x20000000
+        end: 0x20014000
+      is_boot_memory: false
+      cores:
+      - main
+  - !Nvm
+      name: ~
+      range:
+        start: 0
+        end: 0x58000
+      is_boot_memory: true
+      cores:
+      - main
+  flash_algorithms:
+  - cc13x2-cc26x2-flash
+flash_algorithms:
+- name: cc13x2-cc26x2-flash
+  description: Flash algorithm for the cc13x2-cc26x2 family of devices
+  default: true
+  # Generated from https://github.com/seanmlyons22/cc13x2-cc26x2-flash/commit/f793a5a1c6642e84d47c63d1f000ebe91c247480
```

Comment:
Good point. I got eager and finished that too. I have updated the change log.

I also added https://github.com/probe-rs/probe-rs/issues/2215 since at least for my purposes this is resolved, but what I did here is really more of a workaround than what I described in 2215.

That being said, I am no longer sure that 2215 is necessary.